csi/volumeutils.py

```diff
@@ -64,6 +68,80 @@ def get(self):
         """Get Volume name"""
         return self.volname
 
+def connectSocket(client_socket):
+    retry_interval = 60
+    max_retries = 5
+
+    retry_count = 0
+    connected = False
```

Comment:
do we really need connected and is_connected? They're representing the same state...